M2/Macaulay2/packages/Macaulay2Doc/doc_rings.m2

```diff
@@ -136,12 +136,10 @@ document {
 	 (heft, Ring),
 	 (heft, PolynomialRing),
 	 (heft, QuotientRing),
-	 (heft, Module),
-	 (heft, GradedModule),
```

Comment:
Both are removed in favor of `heft ring X`. I kept `(heft, Module)` because some packages use it, but it should probably be deprecated as well. This was following Dan's decision on a previous issue.